### PR TITLE
LongTail:

### PR DESCRIPTION
**Pull request by numismatika**
- add a few more domains

### CODE REVIEW

File: src/chrome/content/rules/LongTail.xml [hunk 1]
```diff
@@ -1,67 +1,89 @@
...
-		<target host="s0-www.ltvimg.com" />
-		<target host="s1-www.ltvimg.com" />
-
-		<target host="insights.jwplayer.com" />
-		<target host="status.jwplayer.com" />
-
-	Timeout/Refused (March 2016):
-		<target host="jw7.jwplayer.com" />
-		<target host="demo.jwplayer.com" />
-		<target host="sdk.jwplayer.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/LongTail.xml]: Please keep this host around. LGTM apart from that.
Please don't overwrite this commit in your branch when fixing this.

Comment (J0WI) [reply src/chrome/content/rules/LongTail.xml]: ping @numismatika

Comment (numismatika) [reply src/chrome/content/rules/LongTail.xml]: sdk got lost in the momentum. Still doesn't respond unfortunately

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**